README.md

```diff
@@ -1,16 +1,124 @@
 # Peer Code Review App
-A SPA paid and credit based peer review platform.
-## Running MongoDB locally
+
+> A web app that matches programmers to a mentor to review their code.
+
+## About
+
+A web app that matches programmers to a mentor to review their code. Programmers earn credits by giving code reviews, and you can spend credits on getting your code reviewed (by someone a level up from you).
+
+### Who’s it for?
+
+Programmers who are looking for a mentor to provide code feedback!
+
+### User Journey
+
+Context: I’m a bootcamp grad and I’m new to the field! To improve my coding skills, I want to get feedback from people more experienced than me to make sure I’m following best practices and learn where I can improve.
+
+I sign on to the platform and create an account. I input my details in my profile (i.e. beginner with <1 year of work experience).
+
+I begin with 3 credits (1 credit = 1 code review)
+
+I upload a snippet of my code with context around what I’m writing to submit for review.
+
+In 3 days I get a notification that someone has reviewed my code and I can review their notes!
```

Comment:
probably do not need to add this part since user will get a notification instantly.

Reply:
Just copied from the spec ;)